src/jackdaw/test/fixtures.clj

```diff
@@ -201,7 +201,7 @@
     (let [rt (StreamsResetter.)
           app-id (get app-config "application.id")
           args (->> ["--application-id" (get app-config "application.id")
-                     "--bootstrap-servers" "localhost:9092"]
+                     "--bootstrap-servers" (or (get app-config "bootstrap-servers") "localhost:9092")]
```

Comment:
Think "bootstrap-servers" here should be "bootstrap.servers" but LGTM apart from that.

Reply:
oh, sorry, I forgot that little detail :) you are correct